sdk/openai/openai/src/models/models.ts

```diff
@@ -226,6 +248,25 @@ export interface FunctionName {
   name: string;
 }
 
+/**
+ *   A representation of configuration data for a single Azure OpenAI chat extension. This will be used by a chat
+ *   completions request that should use Azure OpenAI chat extensions to augment the response behavior.
+ *   The use of this configuration is compatible only with Azure OpenAI.
+ */
+export interface AzureChatExtensionConfiguration {
+  /**
+   *   The label for the type of an Azure chat extension. This typically corresponds to a matching Azure resource.
+   *   Azure chat extensions are only compatible with Azure OpenAI.
+   */
+  type: AzureChatExtensionType;
+  /**
+   *   The configuration payload used for the Azure chat extension. The structure payload details are specific to the
+   *   extension being configured.
+   *   Azure chat extensions are only compatible with Azure OpenAI.
+   */
+  parameters: any;
```

Comment:
could we do better than `any` like `Record<string, any>`?

Reply:
Done in https://github.com/Azure/azure-sdk-for-js/pull/26907/commits/c27052ea439aa224df870352f33555d0d77a53d2.